Resources/views/Core/add_block.html.twig

```diff
@@ -16,58 +16,53 @@
     {% set column_count = (groups|length / items_per_column)|round(0, 'ceil') %}
 
     <div class="dropdown-menu multi-column dropdown-add"
-        {% if column_count > 1 %}style="width: {{ column_count*140 }}px;"{% endif %}
+         {% if column_count > 1 %}style="width: {{ column_count*140 }}px;"{% endif %}
             >
+        {% set flag=false %}
         {% for group in groups|reverse %}
             {% set display = (group.roles is empty or is_granted('ROLE_SONATA_ADMIN') ) %}
             {% for role in group.roles if not display %}
                 {% set display = is_granted(role) %}
             {% endfor %}
 
-            {% if display %}
+            {% if loop.first or loop.index0 % items_per_column == 0 %}
```

Comment:
this could hit twice, couldn't it, what if key 0 isn't the first key in the array? i know its an edge case, but this could bring a lot of headache to a developer